app/views/invitations/index.html.erb

```diff
@@ -38,12 +39,14 @@
           </td>
           <td><%= invitation_status(invitation) %></td>
           <td sorttable_customkey=<%= invitation.created_at %>>Invited <%= time_ago_in_words(invitation.created_at) %> ago</td>
+          <td><%= link_to("expire", expire_invitation_path(invitation, page: params[:page]), method: :put, data: {confirm: "Are you sure you want to mark this invitation as expired?" }) if invitation.pending? %></td>
```

Comment:
```suggestion
          <td><%= link_to("Cancel", expire_invitation_path(invitation, page: params[:page]), method: :put, data: {confirm: "Are you sure you want to mark this invitation as expired?" }) if invitation.pending? %></td>
```

I think this link should say 'Cancel'.

Reply:
Done! (I've not commit the suggestion bc the change also affects the specs)